internal/repositories/rewards.go

```diff
@@ -163,14 +119,136 @@ func (r *Repository) PaginateVehicleEarningsByID(ctx context.Context, vehicleEar
 		startCursor = &sc
```

Comment:
could we reference the edges array made above here?
	var endCur, startCur *string

	if len(all) != 0 {
		startCur = &edges[0].Cursor
		endCur = &edges[len(edges)-1].Cursor
	}